.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -144,6 +144,7 @@ body:
       - internal/tools
       - pkg/batchperresourceattr
       - pkg/batchpersignal
+      - pkg/core
```

Comment:
nitpicking - do you really need `pkg/core/idutils` ? Can you use `pkg/idutils` ?

Reply:
wanted to keep the same structure as it is in `internal/coreinternal` in case more packages will be made public in the future, but sure it's not a problem to change it to `pkg/idutils` if the current state is an issue.